tests/test_import.py

```diff
@@ -28,6 +28,38 @@ def test_import_json(app, status, warning):
     # :collapsed: work
     assert "collapsed_TEST_01" in html
 
+    # Check absolute path import
+    abs_path_import_html = Path(app.outdir, "subdoc/abs_path_import.html").read_text()
+    assert "small_abs_path_TEST_02" in abs_path_import_html
+
+    # Check relative path import
+    rel_path_import_html = Path(app.outdir, "subdoc/rel_path_import.html").read_text()
+    assert "small_rel_path_TEST_01" in rel_path_import_html
+
+    # Check deprecated relative path import based on conf.py
+    deprec_rel_path_import_html = Path(app.outdir, "subdoc/deprecated_rel_path_import.html").read_text()
+    assert "small_depr_rel_path_TEST_01" in deprec_rel_path_import_html
+
+    warnings = warning.getvalue()
+    assert (
+        "WARNING: Deprecation warning: Relative path must be relative to the current document in future, "
```

Comment:
No need to check for the complete warning.
" Deprecation warning:" should be enough.

So we can change the message without changing the tests

Reply:
done.